jenkins/test/testsuite/Jenkinsfile

```diff
@@ -62,26 +59,60 @@ pipeline {
             steps {
                 script {
                     basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"
+                    sh "wget ${basePath}/builds/manifest.yml"
                     sh "./test.sh ${JOB_NAME} ${basePath} --test-run-id ${env.BUILD_NUMBER}"
                 }
             }
             post() {
                 always {
                     script {
-                        publish()
+                        uploadTestResults(
+                            upload: true,
+                            jobName: "bundle_build",
+                            buildNumber: "${build_id}"
+                        )
                     }
                     cleanWs disableDeferredWipeout: true, deleteDirs: true
                 }
             }
         }
     }
-}
+    post() {
+        success {
+            node('Jenkins-Agent-al2-x64-m5xlarge') {
+                script {
+                    def stashed = lib.jenkins.Messages.new(this).get(['integ-test'])
+                    publishNotification(
+                        icon: ':white_check_mark:',
+                        message: 'Integration Tests Successful',
+                        extra: stashed,
+                        credentialsId: 'INTEG_TEST_WEBHOOK',
+                    )
+
+                    cleanWs(
+                        disableDeferredWipeout: true,
+                        deleteDirs: true
+                    )
+                }
+            }
+        }
+        failure {
+            node('Jenkins-Agent-al2-x64-m5xlarge') {
+                script  {
+                    def stashed = lib.jenkins.Messages.new(this).get(['integ-test'])
+                    publishNotification(
+                        icon: ':warning:',
+                        message: 'Failed Integration Tests',
+                        extra: stashed,
+                        credentialsId: 'INTEG_TEST_WEBHOOK',
```

Comment:
Should this be `BUILD_NOTICE_WEBHOOK` as stated [here](https://github.com/opensearch-project/opensearch-build/pull/1105/files#diff-15de42ee50e7efc22a48cb1133f6a05fd4472e5a18fb004259b3cb3b5d11264fR160)

Reply:
No. We have created a separate channel for the tests and it has a different webhook. Check the discussion here https://github.com/opensearch-project/opensearch-build/pull/972#issuecomment-971814828.

Reply:
My bad got confused with `tests` and `test` folder. Cool